live-examples/css-examples/basic-user-interface/outline-width.html

```diff
@@ -0,0 +1,37 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="outline-width">
+
+    <div class="example-choice">
+        <pre><code id="example_one" class="language-css">outline-width: thick;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_two" class="language-css">outline-width: 12px;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_three" class="language-css">outline-width: 1em;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_four" class="language-css">outline-width: 0.5ex;</code></pre>
```

Comment:
Instead of the three different units of measure, I would suggest one using px/em/rem and the other three use the  value keywords, thin, medium, and thick